pkg/serverstate/statetest/statetest.go

```diff
@@ -42,6 +45,20 @@ func Test(t *testing.T, f Factory, rf RestartFactory) {
 					name = name[idx+1:]
 				}
 
+				skip := false
+				for _, skipTest := range skipTests {
+					if name == skipTest {
+						skip = true
+						break
+					}
+				}
+				if skip {
+					t.Run(name, func(t *testing.T) {
+						t.Skip("Test is on the skip list - ignoring")
```

Comment:
Can we print the name too? I think that will be helpful if anyone is ever confused why their test didn't run.